### PR TITLE
Roll pub packages [manual]

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by kevmoo** [comment]
Holy cow we got to roll some OUTDATED stuff w/ this!

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -14,15 +14,15 @@ dependencies:
...
+  dwds: 24.1.0
   completion: 1.0.1
   coverage: 1.9.0
   crypto: 3.0.3
   ffi: 2.1.2
   file: 7.0.0
   flutter_template_images: 4.2.0
   html: 0.15.4
-  http: 0.13.6
+  http: 1.2.2
```
Comment (christopherfujino) [inline packages/flutter_tools/pubspec.yaml:25]: FYI @andrewkolos

Comment (kevmoo) [reply packages/flutter_tools/pubspec.yaml:25]: https://pub.dev/packages/http/versions/0.13.6 - 15 months old!

Comment (christopherfujino) [reply packages/flutter_tools/pubspec.yaml:25]: yep, Andrew and I had chatted about this and were concerned. Thanks for the fix!

**Pull request merged**